website/project/model.py

```diff
@@ -3039,3 +3034,137 @@ def approve_embargo(self, user, token):
                 },
                 auth=Auth(user),
             )
+
+class DraftRegistration(AddonModelMixin, StoredObject):
+
+    is_draft_registration = True
+
+    _id = fields.StringField(primary=True, default=lambda: str(ObjectId()))
+
+    datetime_initiated = fields.DateTimeField(auto_now_add=True)
+    datetime_updated = fields.DateTimeField(auto_now=True)
+
+    branched_from = fields.ForeignField('node')
+
+    initiator = fields.ForeignField('user')
+
+    registration_metadata = fields.DictionaryField({})
```

Comment:
Why is an empty `dict` passed? Is it meant to be the default? If so, don't use a mutable value as the default for a field. Instead, use a callable--in this case, the `dict` constructor.

```python
fields.DictionaryField(default=dict)
```